docs/RELAYING_REWARDS.md

```diff
@@ -0,0 +1,106 @@
+# Relaying rewards
```

Comment:
@taariq Take a look at this when you get a chance please. This doc outlines the situation of how refunds will work once deployed. Let's make sure it's good to go.

Reply:
Whoa. Just missed this. @byte-bandit. Apologies. Will gander.